branding/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./dist/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "yarn && babel src --presets babel-preset-env --out-dir dist"
+    "build": "tsc"
```

Comment:
```suggestion
    "build": "yarn && tsc"
```

Reply:
Let's keep the yarn command here to make sure dependencies are installed before the build